build.gradle

```diff
@@ -22,6 +22,14 @@ subprojects { Project subproject ->
     }
 
     apply plugin: "org.gradle.test-retry"
+
+    dependencies {
+        api "io.micronaut.reactor:micronaut-reactor"
```

Comment:
Add a `TODO` here to remove it when the RxJava2 -> Reactor PR is merged in core

Reply:
You should depend on `reactor-core` not in `micronaut-reactor`. It should be `implementation` not `api`.